### DISCUSSION TITLE
是否有关于Windows平台使用的具体教学？

### DISCUSSION THREAD

**Discussion by sebastian0619** [pengxiao-song/LaWGPT#76] [Q&A（问答）]
这几天一直靠着问GPT在windows尝试部署本项目。但是还是碰到了一些问题。

因为运行命令是bash命令，windows不支持。GPT给了俩方案，一个事用git bash 另一个是用wsl下装个ubuntu

两个都用了，但都碰到问题：
1. Git bash装conda和虚拟环境都很顺畅，但是到运行bash的时候还是出现问题了。gpt给的回复大概就是因为能运行但是git bash本身还是基于windows运行。内部的一些代码需要做一些适应性调整才能再windows使用；
2. WSL运行顺畅，但是cuda识别不到我的3060显卡。gpt解决问题以后，运行bash命令依然提示不能识别到GPU 不知道问题出在哪。然后因为wsl装在系统盘，直接超过硬盘空间，初次运行的15g安装不了。所以重新安装conda，装在D盘。但不知道咋说，wsl读写d盘的速度都特别慢，一个conda装了三四个小时，都有可能失败。
周末两天零零碎碎的时间都浪费在这上面，所以还是想着来项目里请教一下大佬们，谢谢！

**Comment by moye12325** [comment]
无法识别GPU多半是CUDA的安装问题，系统盘不够读写慢等问题只能你自己解决。bash执行的问题你不给出截图没人可以猜到你遇到了什么问题
